src/main/holochainManager.ts

```diff
@@ -64,21 +64,27 @@ export class HolochainManager {
       ? parseInt(process.env.ADMIN_PORT, 10)
       : await getPort();
 
-    const conductorConfig = defaultConductorConfig(
-      adminPort,
-      rootDir,
-      lairUrl,
-      bootstrapUrl,
-      signalingUrl,
-      'kangaroo',
-      false,
-      iceUrls,
-      undefined
-    );
+    const conductorConfig = CONDUCTOR_CONFIG_TEMPLATE;
```

Comment:
We overwrite the config based on the template and optional inputs from CLI args on each startup. Respecting manual changes of the conductor-config.yaml on the filesystem would require more advanced fiddling here which I think would be overkill at this point.

Reply:
> We could just load the existing conductor config if it exists

It will always exist after the first run because we will have generated it. So we can't easily distinguish whether it has been modified or not.

Reply:
But that's fine right? Just load the existing conductor config and overwrite the signal, bootstrap and ice urls? That won't affect any other config that has been changed.

Reply:
I think you're right. I must have bad memories from it being cumbersome in Rust when fiddling with serde_yaml Values.

It does mean though that if you change values manually in the conductor config, you'll need to remember the original values because kangaroo is not gonna reset it to the default values.

Reply:
> It does mean though that if you change values manually in the conductor config, you'll need to remember the original values because kangaroo is not gonna reset it to the default values.

I'm not following

Reply:
I think it's not an important point. I just meant to say that there is a risk to accidentally modify the config manually and then forget the default values and be left with a broken app. But I think anyone playing around with the conductor config on disk should know what they're doing.

Reply:
496c75c68f6326c0d321f4a6306a51e0ff586950